source/core/dimensionality.cc

```diff
@@ -23,6 +23,7 @@ namespace Parameters
     thermal_conductivity_scaling     = 1. / M / L * T * T * T * theta;
     enthalpy_scaling                 = 1. / M / L / L * T * T;
     diffusivity_scaling              = 1. / L / L * T;
+    reaction_constant_scaling        = 1. / T;
```

Comment:
it work only for the first order reaction

Reply:
yeah .... that's true
Do we use the dimensionallity functions a lot? I always felt I was unsure of this feature

Reply:
Yeah I don't this we use is a lot, and like @hepap said we could add an additional parameter for order of the reaction. In this case I would remove the `reaction_constant_scaling` variable.